smqtk_detection/impls/detect_image_objects/resnet_frcnn.py

```diff
@@ -0,0 +1,202 @@
+import numpy as np
+from typing import Tuple, Iterable, Dict, Hashable, List
+import importlib.util
+from types import MethodType
+
+try:
+    import torch  # type: ignore
+    import torchvision.models as models  # type: ignore
+    from torchvision import transforms  # type: ignore
+    from torchvision.ops import boxes as box_ops  # type: ignore
+    import torch.nn.functional as F  # type: ignore
+    from torchvision.models.detection.roi_heads import RoIHeads  # type: ignore
+except ModuleNotFoundError:
+    pass
+
+from smqtk_detection.interfaces.detect_image_objects import DetectImageObjects
+from smqtk_detection.utils.bbox import AxisAlignedBoundingBox
+
+import warnings
+warnings.filterwarnings("ignore", category=UserWarning)
```

Comment:
I must have missed this the first time around: I think this kind of thing would be at the application level, not here in a library module. Otherwise this module is making this decision for all users, which I'm not sure is applicable.